solr/core/src/test/org/apache/solr/cloud/hdfs/HdfsTestUtil.java

```diff
@@ -62,73 +63,88 @@
   public static MiniDFSCluster setupClass(String dir) throws Exception {
     return setupClass(dir, true, true);
   }
-  
+
   public static MiniDFSCluster setupClass(String dir, boolean haTesting) throws Exception {
     return setupClass(dir, haTesting, true);
   }
-  
+
+  /**
+   * Checks that commons-lang3 FastDateFormat works with configured locale
+   */
+  @SuppressForbidden(reason="Call FastDateFormat.format same way Hadoop calls it")
+  private static void checkFastDateFormat() {
```

Comment:
commons-lang3 has an issue with the locale `ja-JP-u-ca-japanese-x-lvariant-JP` but this will catch if there are other locales with issues.